spec/unit/knife/core/bootstrap_context_spec.rb

```diff
@@ -141,7 +141,7 @@
     let(:config) { { policy_name: "my_app_server", policy_group: "staging" } }
 
     it "includes them in the first_boot data and excludes run_list" do
-      expect(Chef::JSONCompat.to_json(bootstrap_context.first_boot)).to eq(Chef::JSONCompat.to_json(config))
+      expect(Chef::JSONCompat.to_json(bootstrap_context.first_boot)).to eq(Chef::JSONCompat.to_json(config.delete(:first_boot_attributes)))
```

Comment:
is this delete necessary because the config hash above doesn't seem to have that key?

Reply:
Thanks @lamont-granquist, Added appropriate expectation.